src/Subscriptions/StorageManager.php

```diff
@@ -100,8 +106,13 @@ public function storeSubscriber(Subscriber $subscriber, string $topic): void
 
         $topic[] = $subscriber->channel;
 
-        $this->cache->forever($topicKey, json_encode($topic));
-        $this->cache->forever($subscriberKey, $subscriber);
+        if ($this->ttl === null) {
```

Comment:
If `$this->ttl === null`, Laravel will store the item in the cache forever anyways. The explicit approach you chose is more obvious though, so i am fine with it.

Reply:
This only works since 5.8, I had the third argument set to `null` but on 5.7 and below that won't work since the argument there is not nullable and would result in a ttl of `0` meaning it expires immediately (https://github.com/laravel/framework/blob/5.5/src/Illuminate/Cache/RedisStore.php#L90).